crypto/evp/m_sigver.c

```diff
@@ -279,7 +280,15 @@ static int do_sigver_init(EVP_MD_CTX *ctx, EVP_PKEY_CTX **pctx,
     if (ctx->pctx->pmeth->digest_custom != NULL)
         ctx->pctx->flag_call_digest_custom = 1;
 
-    return 1;
+    ret = 1;
+
+ end:
+#ifndef FIPS_MODULE
+    if (ret > 0)
+        ret = evp_pkey_ctx_unleash_cached_data(locpctx);
```

Comment:
use would probably be better than unleash..

Reply:
Meh...

yeahok

Reply:
Done

Reply:
either this or unleash_dragons()

Reply:
Hahaha, don't tempt me 😉